exporter/clickhouseexporter/README.md

```diff
@@ -382,6 +392,9 @@ service:
       exporters: [ clickhouse ]
 ```
 
+**Note:**
+Parameter `metrics_table_name` will be deprecated and replaced by `metric_tables` which support setting the full table name (including suffix) for tables of each metric type. Using `metrics_table_name` and `metric_tables` together will result in ignoring the `metrics_table_name` parameter value.
```

Comment:
> Parameter `metrics_table_name` will be deprecated

I would say it *is* deprecated as part of this change. Also I would add this table config information to the "configuration options" section. You can remove `metrics_table_name` from that section.

It looks like we removed `ttl_days` too, so our precedent for deprecating fields is to remove them from the README. So we don't need this note at all, my mistake for suggesting it in the first review